iamclient/iamclient.go

```diff
@@ -139,6 +154,51 @@ func (client *Client) GetSystemID() (systemID string) {
 	return client.systemID
 }
 
+// GetNodeInfo returns node info.
+func (client *Client) GetNodeInfo(nodeID string) (nodeInfo cloudprotocol.NodeInfo, err error) {
+	log.WithField("nodeID", nodeID).Debug("Get node info")
+
+	ctx, cancel := context.WithTimeout(context.Background(), iamRequestTimeout)
+	defer cancel()
+
+	request := &pb.GetNodeInfoRequest{NodeId: nodeID}
+
+	response, err := client.nodeService.GetNodeInfo(ctx, request)
+	if err != nil {
+		return cloudprotocol.NodeInfo{}, aoserrors.Wrap(err)
+	}
+
+	return pbconvert.NewNodeInfoFromPB(response), nil
+}
+
+// GetAllNodeIDs returns node ids.
+func (client *Client) GetAllNodeIDs() (nodesId []string, err error) {
+	log.Debug("Get all node ids")
+
+	ctx, cancel := context.WithTimeout(context.Background(), iamRequestTimeout)
+	defer cancel()
+
+	response, err := client.nodeService.GetAllNodeIDs(ctx, &emptypb.Empty{})
+	if err != nil {
+		return nil, aoserrors.Wrap(err)
+	}
+
+	return response.Ids, err
+}
+
+// SubscribeNodeInfoChange subscribes client on NodeInfoChange events
+func (client *Client) SubscribeNodeInfoChange() <-chan cloudprotocol.NodeInfo {
+	client.Lock()
+	defer client.Unlock()
+
+	log.Debug("Subscribe on NodeInfoChanged event")
```

Comment:
Do not use function name or variable names in log

Reply:
done